extension/WebJobs.Extensions.RabbitMQ/Trigger/RabbitMQListener.cs

```diff
@@ -20,6 +21,9 @@ namespace Microsoft.Azure.WebJobs.Extensions.RabbitMQ
 {
     internal sealed class RabbitMQListener : IListener, IScaleMonitor<RabbitMQTriggerMetrics>
     {
+#pragma warning disable SA1000
+        private static readonly ActivitySource Source = new("Microsoft.Azure.WebJobs.Extensions.RabbitMQ");
```

Comment:
>Source [](http://example.com/codeflow?start=47&length=6)

`Source` is too generic to be used as class member variable name. Rename it to `ActivitySource`.